src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -94,12 +94,6 @@ def __init__(self):
         self._ffi = self._binding.ffi
         self._lib = self._binding.lib
 
-        # Set the default string mask for encoding ASN1 strings to UTF8. This
```

Comment:
Will this cause problems for pyOpenSSL?

Reply:
pyOpenSSL actually does this independently (which makes sense since this is in the backend, which is only imported in pyOpenSSL under certain conditions). See: https://github.com/pyca/pyopenssl/blob/9e15eca29625f0aa87635a613b093ba1ecaa28fb/src/OpenSSL/crypto.py#L3035